app/js/controllers/controller-outfit.js

```diff
@@ -591,6 +595,30 @@ angular.module('app').controller('OutfitController', ['$window', '$rootScope', '
         }
       }
     }
+
+    //limpets if controllers exist and cargo space available
+    if (limpets > 0) {
+      item = {
+        ammoName: 'limpets',
+        ammoMax: ship.cargoCapacity,
+        ammoUnitCost: 101,
+        ammoTotalCost: ship.cargoCapacity * 101
+      };
+      costs.push(item);
+      total += item.ammoTotalCost;
+    }
+    //total fuel, or if slider isn't at max, use that value and only if scoop not present
+    if (!scoop) {
+      q = $scope.fuel != ship.fuelCapacity ? $scope.fuel : ship.fuelCapacity;
```

Comment:
Yeah. I think when I initially tried it, I hadn't put in the updateAmmoCosts to fuelChange so it showed up as zero. Lemme clean that up and commit.